trestle/core/remote/cache.py

```diff
@@ -278,6 +280,8 @@ def _do_fetch(self) -> None:
                     raise TrestleError(f'Cache update failure with bad inputenv var: {err_str}')
         if self._username is not None and self._password is not None:
             auth = HTTPBasicAuth(self._username, self._password)
+
+        # attempt get of the file from url
```

Comment:
Comment is not necessary here

Reply:
removed - but it's there because previously it appeared linked to the password query, while this get() is generic.